### PR TITLE
test

### PR DESCRIPTION
**Pull request by Davidmtw2**

### CODE REVIEW

File: HotelProject/src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -3,5 +3,6 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        System.out.println("hi");
```
Comment (osbokirk) [inline HotelProject/src/main/java/org/example/Main.java:6]: "Hi" should be capitalized

Comment (ahmawad) [inline HotelProject/src/main/java/org/example/Main.java:6]: WOWZERZ

**Review by osbokirk — APPROVED** [review]

File: HotelProject/src/main/java/org/example/Main.java (hunk 1 above)
Comment (DigitalGriot) [reply HotelProject/src/main/java/org/example/Main.java:6]: i c u

Comment (DigitalGriot) [reply HotelProject/src/main/java/org/example/Main.java:6]: No!

**Pull request merged**